wallet-new/server/Cardano/Wallet/API/V1/LegacyHandlers/Accounts.hs

```diff
@@ -64,3 +66,12 @@ updateAccount wId accIdx accUpdate = do
     accMeta <- migrate accUpdate
     cAccount <- V0.updateAccount newAccId accMeta
     single <$> (migrate cAccount)
+
+
+newExternalAccount
+    :: (V0.MonadWalletLogic ctx m)
+    => WalletId
+    -> NewAccount
+    -> m (WalletResponse Account)
+newExternalAccount _ _ = do
+    single <$> (liftIO $ generate arbitrary)
```

Comment:
I strongly prefer *not* to have type-checking and warning-free code that is dramatically wrong. Let's replace this with `undefined` for now. That way we'll get a warning and CI will fail to build, which will be a good way to keep this out of `develop` :)